apis/nucleus/src/components/ActionsToolbar.jsx

```diff
@@ -110,6 +106,10 @@ function ActionsToolbar({
   },
   focusHandler = null,
   actionsRefMock = null, // for testing
+  popoverAnchorOrigin = {
+    vertical: 'top',
+    horizontal: 'right',
+  },
```

Comment:
Why move this code?
And why the theme spacing change below it?
This file is used by all charts, so rather not make changes unless there is a good reason.

Reply:
Its moved to be able to override the defaults from ListboxInline.
This is to align with placement and looks in sense-client.

Reply:
Can we detect if we are in sense-client, and only apply these changes then?

Reply:
Where are you doing this override? and what is the diff between client and outside?